packages/ui-extensions-react/src/surfaces/customer-account/hooks/live-navigation.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  NavigationHistoryEntry,
+  RenderCustomerAccountFullPageExtensionTarget,
+} from '@shopify/ui-extensions/src/surfaces/customer-account';
+import {useApi} from './api';
+import {useEffect, useReducer} from 'react';
+
+export function useLiveFullPageNavigation<
```

Comment:
Extension points without full page target cant use this api

Reply:
Just did! If its a standard navigation, we wont have the current entry and the navigation events so throwing an error in that case